packages/ckeditor5-ui/src/spinner/spinnerview.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/spinner/spinnerview
+ */
+
+import { type Locale } from '@ckeditor/ckeditor5-utils';
+
+import View from '../view';
+
+import '../../theme/components/spinner/spinner.css';
+
+/**
+ * The spinner view class.
+ */
+export default class SpinnerView extends View {
+	/**
+	 * Controls whether the spinner is visible.
+	 *
+	 * @observable
+	 * @default false
+	 */
+	declare public isVisible: boolean;
+
+	/**
+	 * @inheritDoc
+	 */
+	constructor( locale?: Locale ) {
```

Comment:
I'm not sure if `Locale` will be of any use for this view ever. There's nothing to translate here and it's unlikely to get any children that will need translations.